tests/tests.rs

```diff
@@ -48,7 +48,9 @@ fn test_valid(input: &[u8]) {
     test_valid_public_imp(input);
 }
 
+// unused for cases where public_imp is set but no SSE functions generated...
```

Comment:
I think "SSE" should be "SIMD" here and in all the following comments?

Reply:
Good catch--will fix.